ios/Classes/Bindings/SdkBinding.swift

```diff
@@ -34,8 +34,12 @@ class SdkBinding: Binding {
         completionHandler: FlutterMethodCallCompletionHandler
     ) {
         do {
+            let accessToken: String? = try flutterArguments.asDictionary(argKey: "accessToken").decode()
```

Comment:
I think we can decode this as non optional. Also we should modify the dart file.

Reply:
updated